mergify_engine/mergify_pull.py

```diff
@@ -120,16 +132,16 @@ def complete(self, cache, **context):
             self._approvals = self._compute_approvals(**context)
 
         if "mergify_engine_required_statuses" in cache:
-            self._required_statuses = cache[
-                "mergify_engine_required_statuses"]
+            self._required_statuses = StatusState(cache[
```

Comment:
Maybe convert old cache integer to the string (and put need_to_be_saved to True) ? or do you prefer I refresh the cache of all installations once it's merged ?

Reply:
Since I'd prefer to keep it simple I'd vote for clearing the cache, but if you think it's a burden, I can be convinced otherwise.

Reply:
Looks like we didn't hit the rate limit anymore, I will refresh the cache